app/frontend/schools/settings/components/SchoolCustomize__LinkComponent.res

```diff
@@ -127,18 +127,29 @@ module LinkEditor = {
       {switch kind {
       | HeaderLink
       | FooterLink => <>
-          <input
-            value=state.title
-            required=true
-            autoFocus=true
-            id={"link-title-" ++ id}
-            className=inputClasses
-            placeholder="A short title for a new link"
-            onChange={event => {
-              let value = ReactEvent.Form.target(event)["value"]
-              send(UpdateTitle(value))
-            }}
-          />
+          <div className="flex flex-col gap-1 flex-1">
+            <input
+              value=state.title
+              required=true
+              autoFocus=true
+              id={"link-title-" ++ id}
+              className=inputClasses
+              placeholder="A short title for a new link"
+              onChange={event => {
+                let value = ReactEvent.Form.target(event)["value"]
+                send(UpdateTitle(value))
+              }}
+              maxLength=24
+            />
+            <School__InputGroupError
+              active={!StringUtils.isPresent(state.title)} message={t("cant_empty_message")}
+            />
+            <School__InputGroupError
+              active={state.title->String.length == 24}
+              message={t("reached_max_title_length")}
+              warn={true}
+            />
```

Comment:
@pupilfirst/designers whats your thoughts on showing two error messages? 

Also @yash-learner do you think we need to show an error message for empty `title`, a smart min and max message should be enough right?

Reply:
> Also @yash-learner do you think we need to show an error message for empty title, a smart min and max message should be enough right?

Yes, but error message for empty and a separate warning message when text length reaches `24` characters make it much clearer and user-friendly but with a smart min and max can do it.

----
I had done this in validator 

I had extracted the two validators in the validator into a single one.

Reply:
@yash-learner @bodhish 
It's a good idea to include a single error message that covers both the minimum and maximum character limits for the input, as Bodhi suggested.
For example:
`Please enter a title between 3 and 24 characters in length.`
And let's make sure the error message maintains a friendly tone. 
Instead of displaying `is not a valid URL`, a user-friendly error message would be `Please enter a valid URL.`
cc @irajsuhail